catalog.yml

```diff
@@ -178,3 +178,7 @@
 - repo_url: https://github.com/greenelab/phenoplier_manuscript
   html_url: https://greenelab.github.io/phenoplier_manuscript/
   preprint_citation: doi:10.1101/2021.07.05.450786
+- repo_url: https://github.com/greenelab/mpmp-manuscript
+  html_url: https://greenelab.github.io/mpmp-manuscript/
+  thumbnail_url: https://user-images.githubusercontent.com/2345877/140061720-96b5451a-c9af-493f-85b2-cc4c723d316b.png
```

Comment:
Any reason not to [define the thumbnail](https://github.com/manubot/rootstock/blob/9a448fe6edb477fb98065d5b83d206b1ca489cee/USAGE.md#thumbnail) at the manuscript level so it also works for social media shares?

Reply:
I removed `thumbnail_url` and will merge now! Whenver the manuscript adds the thumbnail and CI rebuilds the catalog, then the record for this manuscript will include the thumbnail

Reply:
Sounds good! Working on adding a properly formatted thumbnail in greenelab/mpmp-manuscript#37.